tox.ini

```diff
@@ -18,6 +18,7 @@ deps =
     coveralls: coveralls
     py33,py34: -r{toxinidir}/requirements.txt
     py27: -r{toxinidir}/requirements-py2.txt
+passenv = DOCKER_HOST DOCKER_CERT_PATH DOCKER_MACHINE_NAME DOCKER_TLS_VERIFY DOCKER_IP
```

Comment:
It turns out that most of my problems were just not realizing we had to white list necessary env in tox.ini even though the bash script runs with `#!/usr/bin/env bash`